### PR TITLE
Last Updates on the code as you request 

### PR DESCRIPTION
**Pull request by MaisHasibaTestitg**
Add more logging for case,  remove template default logging (before each / after each/ this is result)  , and reduce code duplication in get title

### CODE REVIEW

**Comment by MaisHasibaTestitg** [comment]
Last Updates on the code

File: Regression/Page/Title.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  assert
+} = require('chai').assert
+const {
+  expect
+} = require('chai')
+
+
+exports.getTitle = (browser) => {
+  var login = browser.page.loginSelectors();
+  login
```
Comment (ibatta) [inline Regression/Page/Title.js]: browser.

File: Regression/Page/Flows.js [hunk 2]
```diff
@@ -0,0 +1,230 @@
+const FlowsSelector =require("../Selectors/FlowsSelector");
+const { assert } = require('chai').assert
+const { expect } = require('chai')
+const configrationReader = require("../utils/configrationReader");
+
+//create new flow
+exports.CreateNewFlow = (browser) =>{
+    var Flows = browser.page.FlowsSelector();
```
Comment (ibatta) [inline Regression/Page/Flows.js]: please try to use a browser instead

File: Regression/tests/MainNavigationBarTests/MainNavigationBarTests.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+   after(function (browser, done) {
+    browser.end(function () {
+        console.log("End Browser");
+        done();
+    });
+});
+  
+beforeEach(function (browser, done) {
+    logReport.log(this, "before each test case : open the site");
```
Comment (ibatta) [inline Regression/tests/MainNavigationBarTests/MainNavigationBarTests.js]: general place

**Pull request merged**